books/projects/curve25519/ecp.lisp

```diff
@@ -106,7 +106,7 @@
   :hints (("Goal" :use ((:instance mod*rewrite-2 (a (* a b)) (b c))))))
 
 (defthm mod*rewrite-4
-  (implies (and (integerp a) (integerp b) (integerp c))
+  (implies (and (integerp a) (integerp b) (integerp c) (integerp d))
```

Comment:
ACL2(r) starts to certify this book after the change. It fails without this hypothesis.